flink-runtime/src/main/java/org/apache/flink/runtime/deployment/ResultPartitionDeploymentDescriptor.java

```diff
@@ -92,6 +92,14 @@ public boolean sendScheduleOrUpdateConsumersMessage() {
 		return sendScheduleOrUpdateConsumersMessage;
 	}
 
+	/**
+	 * Returns whether to release the partition after having been fully consumed once.
+	 *
+	 * <p>Indicates whether the shuffle service should automatically release all partition resources after
+	 * the first full consumption has been acknowledged.
+	 *
+	 * @return whether to release the partition after having been fully consumed once.
```

Comment:
It has the same information with above `Returns whether to release the partition after having been fully consumed once.`, so might be removed.

Reply:
I wanted to keep jdoc format consistent.